src/Microsoft.Graph/Generated/requests/GraphServiceClient.cs

```diff
@@ -29,6 +31,20 @@ public partial class GraphServiceClient : BaseClient, IGraphServiceClient
         {
         }
 
+        /// <summary>
+        /// Instantiates a new GraphServiceClient using a <see cref="TokenCredential"/> instance.
+        /// </summary>
+        /// <param name="tokenCredential">The <see cref="TokenCredential"/> to use for authentication</param>
+        /// <param name="scopes">Scopes required to access Microsoft Graph. This defaults to https://graph.microsoft.com/.default when none is set.</param>
+        /// <param name="httpProvider">The <see cref="IHttpProvider"/> for sending requests.</param>
+        public GraphServiceClient(
+            TokenCredential tokenCredential,
+            IEnumerable<string> scopes = null,
+            IHttpProvider httpProvider = null)
+            : this("https://graph.microsoft.com/beta", new TokenCredentialAuthProvider(tokenCredential,scopes), httpProvider)
```

Comment:
```suggestion
            : this("https://graph.microsoft.com/beta", new TokenCredentialAuthProvider(tokenCredential ?? throw new ArgumentNullException(nameof(tokenCredential)), scopes), httpProvider)
```

Reply:
The Auth provider will set the scopes to be a default value if not provided therefore no need to throw. See [here](https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/bcbc09cbe9b5e4be80f3a39acbd1cd5361257a81/src/Microsoft.Graph.Core/Authentication/TokenCredentialAuthProvider.cs#L36).
Furthermore, the XML comments of the constructor capture this as well.